src/codec/h264/parser.rs

```diff
@@ -717,6 +713,20 @@ pub struct Sps {
 }
 
 impl Sps {
+    /// Returns the coded width of the stream.
+    ///
+    /// See 7-13 through 7-17 in the specification.
+    pub fn width(&self) -> u32 {
```

Comment:
> Doing so will forbid us from using u32::from instead of as u32, which is less error-prone (on paper at least). Do you have any benefit in mind for making these const?

Not really. It's upcasting here so we should be fine. However let's leave it as is.